src/Symfony/Component/Mime/Tests/Header/DateHeaderTest.php

```diff
@@ -35,7 +35,7 @@ public function testDateTimeCanBeSetBySetter()
 
     public function testDateTimeIsConvertedToImmutable()
     {
-        $dateTime = new \DateTime();
+        $dateTime = new \DateTimeImmutable();
```

Comment:
This should be reverted. This test is precisely testing that passing a DateTime as argument gets converted to a DateTimeImmutable.